test/orm/dml/test_update_delete_where.py

```diff
@@ -2585,6 +2585,27 @@ def test_update_from_multitable_same_names(self):
             ],
         )
 
+    def test_delete_args(self):
```

Comment:
this test should probably be right underneath test_update_args in ExpressionUpdateTest